tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -623,10 +599,9 @@ def test_setting_dynamic_sampling_rules_roundtrip(self):
         Tests that we get the same dynamic sampling rules that previously set
         """
         with Feature({"organizations:filters-and-sampling": True}):
-            resp = self.client.put(self.path, data={"dynamicSampling": _dyn_sampling_data()})
-            assert resp.status_code == 200, resp.content
```

Comment:
this one seems like it's missing a replacement

Reply:
Thanks! I looks like there was a bug that was making this test still pass.